plugins/pomodoro-timer/manager.py

```
@@ -0,0 +1,1256 @@
+        if self.timer_thread and self.timer_thread.is_alive():
+            self.timer_thread.join(timeout=2.0)
+        self.timer_thread = None
+
+        if self.mqtt_thread and self.mqtt_thread.is_alive():
+            self.mqtt_stop_event.set()
+            if self.mqtt_client:
+                try:
+                    self.mqtt_client.loop_stop()
+                    self.mqtt_client.disconnect()
+                except Exception:
+                    pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)